package.json

```diff
@@ -29,7 +29,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "jest --coverage",
```

Comment:
we need a separate script for the coverage, can't touch this one it's used for testing and on the CI as well.

Reply:
fixed